rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -577,6 +577,9 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 	u32 errDialog, PSetList setList, PSetBuf setBuf, PFuncList funcList, PFuncFixed funcFixed, PFuncStat funcStat,
 	PFuncFile funcFile, u32 container, u32 unk_op_flags /*TODO*/, vm::ptr<void> userdata, u32 userId, PFuncDone funcDone)
 {
+	if (setList) cellSaveData.notice("savedata_op(): setList = { .sortType=%d, .sortOrder=%d, .dirNamePrefix='%s' }", setList->sortType, setList->sortOrder, setList->dirNamePrefix);
```

Comment:
You need to use vm::ptr<>::try_read() member function here, noone said they are valid pointers. On real ps3 if other args are invalid they might not be dereferenced at all and not crash the application due to early return.

Reply:
I don't think we ever used it anywhere else for such occasions before.
At least it's new to me.

Reply:
if (CellSaveDataSetList list{}; setList.try_read(list))